pypeclub/OpenPype
# Maya: Fix animated attributes (ie. overscan) on loaded cameras breaking review publishing.

## Brief description
Fixes #2807 by checking for locked attributes and skipping them.

## Description
Issue #2807 stems from published cameras being unable to distinguish animated attributes on them when they're published as alembics, so when they're loaded in again, the attributes are automatically connected to the alembic nodes.  This solves that issue by testing if the attribute is locked and passes changing those attributes if they're connected.

## Additional info
It's mainly handled in `capture.py`, with the soft fail wrapper by @kalisp included in case Maya acts up. 

## Testing notes:
1. In Maya, animate attribute on a created camera shape.
2. Create camera instance.
3. Publish camera instance.
4. Load published camera in as alembic.
5. Create review instance from published camera.

Should run smoothly as opposed to breaking during the extract playblast stage.
